src/core/segment_buffers/implementations/text/html/text_track_cues_store.ts

```diff
@@ -72,6 +89,19 @@ export default class TextTrackCuesStore {
             ret.push(cues[j].element);
           }
         }
+        // first or last IHTMLCue in a group can have a slighlty different start
+        // or end time than the start or end time of the ICuesGroup due to parsing
+        // approximation.
+        // Add a tolerance of 1ms to fix this issue
+        if (ret.length === 0 && cues.length) {
```

Comment:
IMO we should still be explicit about the length check here. I don't know why the linter isn't failing here.

Reply:
I have changed it, linter was fine with both ways